_assets/stylesheets/main.scss

```diff
@@ -27,6 +27,7 @@
 @import 'base/page';
 @import 'base/typography';
 @import 'base/syntax';
+@import 'base/variables';

```

Comment:
No longer needed. Remove this line, and delete `_assets/stylesheets/base/_variables.scss`

Reply:
done :)